config.yml.example

```diff
@@ -0,0 +1,7 @@
+port: 5300
+httpPort: 4000
+logLevel: info
+
+upstream:
+  default:
+    - 8.8.8.8
```

Comment:
Am not sure if this is the most appropriate default settings to get started but it seemed like the minimum 🙂

Reply:
I had google servers in the first versions of blocky in the config file. Many people were "not amused", because google is evil  ;) I think we should use only free and uncensored DNS public server in any descriptions/config files. See docs/config.yaml